pkg/handlers/openapi/generate.go

```diff
@@ -0,0 +1,102 @@
+package openapi
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/aserto-dev/aserto/pkg/cc"
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+// Test with: https://github.com/splunk/splunk-cloud-sdk-go/tree/master/services/action
+
+type GenerateOpenAPI struct {
+	Path string `arg:"" required:"" help:"path to openapi.yaml"`
+	Name string `arg:"" required:"" help:"path to openapi.yaml"`
+}
+
+func parseURI(uri string) []string {
+	result := []string{}
+	parts := strings.Split(uri, "/")
+	for _, part := range parts[1:] {
+		if strings.Contains(part, "{") {
+			var clean = strings.Replace(strings.Replace(part, "{", "", -1), "}", "", -1)
+			result = append(result, "__"+clean)
+		} else {
+			result = append(result, part)
+		}
+	}
+	return result
+}
+
+func generatePackageName(root, verb, uri string) string {
+	parts := []string{root, verb}
+	parts = append(parts, parseURI(uri)...)
+	return strings.Join(parts, ".")
+}
+
+func (cmd *GenerateOpenAPI) Run(c *cc.CommonCtx) error {
+
+	specURL, err := url.Parse(cmd.Path)
+	if err != nil {
+		log.Fatal("Failed to parse spec URL", err)
+	}
+	root := cmd.Name
+
+	doc, err := openapi3.NewLoader().LoadFromURI(specURL)
+
+	if err != nil {
+		log.Fatal("Failed to load spec from URL", err)
+	}
+
+	paths := doc.Paths
+
+	var packages []string
```

Comment:
It feels inconsistent to be using `var` in some places and `:=` in others.
I don't think `var` is necessary anywhere in this function. You're just initializing local variables. I'd stick with `:=`.

e.g.
```
packages := []string{}

...

policyDir := root + "/src/policies"
```
etc.